src/index.html

```diff
@@ -13,9 +13,41 @@
       "
     >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img class="card__img" 
+      src="images/imac.jpeg" 
+      alt="Apple A1419 iMac 27 inch Retina 5K Monoblock" 
+      height="134" 
+      width="160"
+      >
```

Comment:
Check the code style. And don't write inline styles, use CSS for it
```suggestion
      <img 
        class="card__image" 
        src="images/imac.jpeg" 
        alt="Apple A1419 iMac 27 inch Retina 5K Monoblock" 
      >
```
<img width="1020" alt="image" src="https://user-images.githubusercontent.com/99403198/234527696-567b1424-302b-485b-9bee-18b09848800e.png">